src/frontend/src/flows/authenticate/index.ts

```diff
@@ -46,8 +47,8 @@ const pageContent = (
   const onRecoverClick = () => useRecovery(connection, readUserNumber());
 
   const authorizeButton = createRef();
-  const userNumberInput = createRef();
   const registerButton = createRef();
+  const anchorInput = mkAnchorInput("userNumberInput", userNumber);
```

Comment:
for fun

```
const {template, refs: {userNumberInput}} = mkAnchorInput("userNumberInput", userNumber);
```

Reply:
I think `ts` would then complain about `template` being bound again...